Ladybird/main.cpp

```diff
@@ -110,8 +110,13 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
     window.resize(800, 600);
     window.show();
 
-    if (auto url = get_formatted_url(raw_url); url.is_valid())
+    if (auto url = get_formatted_url(raw_url); url.is_valid()) {
         window.view().load(url);
+    } else {
+        auto maybe_home_url = ak_string_from_qstring(s_settings->homepage());
+        if (!maybe_home_url.is_error())
+            window.view().load(get_formatted_url(maybe_home_url.release_value().bytes_as_string_view()));
```

Comment:
- The OOM error should be propagated via `TRY()`
- What happens if `s_settings->homepage()` is empty?